website/src/edit-word-panel.css.tsx

```diff
@@ -0,0 +1,52 @@
+import { style } from "@vanilla-extract/css"
+import { button, cleanButton } from "src/components/button.css"
+import { fonts, hspace, radii, vspace } from "src/style/constants"
+import { marginX, paddingX, paddingY } from "src/style/utils"
+
+export const formStyle = style({
```

Comment:
Stylistically, I prefer not having a `Style` suffix and instead importing this file as `css` or even `style(s)` if you like that better. Then you could do `style.form` or `css.button` instead.